Sources/TuistDependenciesTesting/DependenciesGraph/DependenciesGraph+TestData.swift

```diff
@@ -75,6 +75,42 @@ public extension TuistCore.DependenciesGraph {
                                     ],
                                     tags: []
                                 ),
+                                .glob(
+                                    pattern: "\(packageFolder.pathString)/customPath/**/*.xib",
+                                    excluding: [
+                                        "\(packageFolder.pathString)/customPath/excluded/sources/**"
+                                    ]
+                                ),
+                                .glob(
+                                    pattern: "\(packageFolder.pathString)/customPath/**/*.storyboard",
+                                    excluding: [
+                                        "\(packageFolder.pathString)/customPath/excluded/sources/**"
+                                    ]
+                                ),
+                                .glob(
+                                    pattern: "\(packageFolder.pathString)/customPath/**/*.xcdatamodeld",
+                                    excluding: [
+                                        "\(packageFolder.pathString)/customPath/excluded/sources/**"
+                                    ]
+                                ),
+                                .glob(
+                                    pattern: "\(packageFolder.pathString)/customPath/**/*.xcmappingmodel",
+                                    excluding: [
+                                        "\(packageFolder.pathString)/customPath/excluded/sources/**"
+                                    ]
+                                ),
+                                .glob(
+                                    pattern: "\(packageFolder.pathString)/customPath/**/*.xcassets",
+                                    excluding: [
+                                        "\(packageFolder.pathString)/customPath/excluded/sources/**"
+                                    ]
+                                ),
+                                .glob(
+                                    pattern: "\(packageFolder.pathString)/customPath/**/*.lproj",
+                                    excluding: [
+                                        "\(packageFolder.pathString)/customPath/excluded/sources/**"
+                                    ]
+                                )
```

Comment:
Maybe we coulde create a test function so this (and the ones below becomes something like
```suggestion
                            resources: [
                              // custom resources
                            ] + Self.defaultResources(folder: "\(packageFolder.pathString)/Sources/TuistKit/", excluding: "XXX"),
```